Makefile

```diff
@@ -24,7 +24,8 @@ image-push:
 
 .PHONY: test
 test:
-	go test -v $(go list ./... | grep -v e2e)
+	# go test -v $(go list ./... | grep -v e2e)
+	go test -v  ./...
```

Comment:
If I'm not mistaken, I think we only have tests in certification atm. Can we just make this ./certification/...?

Reply:
This also fixes the problem:

	go test -v $(shell go list ./... | grep -v e2e)